user_guide_src/source/libraries/sessions.rst

```diff
@@ -178,9 +178,8 @@ you need it.
 You can simply assign data to the ``$_SESSION`` array, as with any other
 variable. Or as a property of ``$session``.
 
-Alternatively, the old method of assigning it as "userdata" is also
-available. That however passing an array containing your new data to the
-``set_userdata()`` method::
+As opposed to the old doc, userdata is depreciated
```

Comment:
I believe the appropriate word is `deprecated`, not `depreciated`. Either way, though, I try not to reference old docs in current docs. Please rewrite to ignore the old and just let them know how to do it now.